common/src/IO/ObjParser.cpp

```diff
@@ -0,0 +1,252 @@
+/*
+ Copyright (C) 2010-2017 Kristian Duske
+
+ This file is part of TrenchBroom.
+
+ TrenchBroom is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ TrenchBroom is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with TrenchBroom. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "ObjParser.h"
+
+#include "Logger.h"
+#include "Assets/EntityModel.h"
+#include "Assets/Texture.h"
+#include "IO/FileSystem.h"
+#include "IO/FreeImageTextureReader.h"
+#include "IO/Path.h"
+#include "IO/Quake3ShaderTextureReader.h"
```

Comment:
I think this include can be removed, too.

Reply:
Resolved, but I'm unsure when.